pyvista/plotting/plotter.py

```diff
@@ -6885,6 +6885,24 @@ def add_cursor(
 
         return actor
 
+    @property
+    def meshes(self):
+        """Return plotter meshes.
+
+        _extended_summary_
+
+        Returns
+        -------
+        List[Union[pv.PolyData, pv.UnstructuredGrid, pv.UniformGrid]
```

Comment:
Instead of an incomplete list, I would use `pyvista.DataSet` instead.  What is returned with a MultiBlock mesh?  I assume that object?  If so, then it would be `List[Union[pyvista.DataSet, PyVista.MultiBlock]]`.

Reply:
This is something I didn't know at the time and I need help with. I wrote that as my best guess.

I will change it to `pyvista.Dataset`. Thank you!